### ISSUE TITLE
Frozen in place after second arena fight, couldn't get out of typing mode, reloading from main menu put me back in the prologue

### ISSUE THREAD

**Issue by kantra19** [FishingCactus/NanotaleEarlyAccessBugs#29]
**Describe the bug**
After the arena fight ended, I was frozen in place in keyboard mode. I could type, but I couldn't move or exit keyboard mode. I forgot about the reload option for softfreezing, so I went back to the main menu to reload. When I was put back in the game, I was back at the start at the game, I think in the second tutorial.

**To Reproduce**
1. Beat the second arena I guess.
2. If you are frozen, quit to Main Menu.
3. Select Start from the main menu.

**Expected behavior**
Not be frozen after the arena fight is over. The ability to restart from my nearest save from the main menu, which should have been after the arena fight.

**Screenshots**
![20191025211312_1](https://user-images.githubusercontent.com/56983552/67612260-0f33e300-f76f-11e9-9dfc-0b66a4271548.jpg)

**Comment by kantra19** [comment]
As an update, this happened again after I deleted my save and started the game over. However, this time when I was stuck in keyboard mode, I used the respawn button. I respawned and could move, but now I can't enter keyboard move and the camera is frozen to the area I respawned in (which is the same as what's in the second screenshot, sans keyboard mode). So I can't leave this screen. I exited to the main menu, then clicked Start, and the game has restarted from the beginning.
